src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantDatum.java

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.hellbender.tools.walkers.vqsr;
+
+import org.broadinstitute.hellbender.utils.GenomeLoc;
+
+import java.util.Comparator;
+import java.util.List;
+
+/*
+ * Represents a data item for VQSR (a site).
+ * Package private because it's not usable outside of VQSR.
+ */
+final class VariantDatum {
+
+    public double[] annotations; //values of the annotations
+    public boolean[] isNull;     //is any of these values "null" (empty) - used for marginalization for plotting
+    public boolean isKnown;      // is this a known site
+    public double lod;            //lod score from the model
+    public boolean atTruthSite;
+    public boolean atTrainingSite;
+    public boolean atAntiTrainingSite;
+    public boolean isTransition;
+    public boolean isSNP;
+    public boolean failingSTDThreshold;
+    public double prior;                 //per-site prior
+    public GenomeLoc loc;
+    public int worstAnnotation;
+    public boolean isAggregate; // this datum was provided to aid in modeling but isn't part of the input callset
```

Comment:
Non-final public fields should ideally be either properly encapsulated or made final.